zulipterminal/ui_tools/boxes.py

```diff
@@ -41,11 +41,11 @@ def __init__(self, view: Any) -> None:
         self.recipient_user_ids = []  # type: List[int]
         self.msg_body_edit_enabled = True
         self.FOCUS_HEADER_CONTAINER = 0
-        self.FOCUS_HEADER_STREAM_BOX = 0
-        self.FOCUS_HEADER_TOPIC_BOX = 1
-        self.FOCUS_HEADER_EDIT_BOX = 2
+        self.FOCUS_HEADER_STREAM_BOX = 1
+        self.FOCUS_HEADER_TOPIC_BOX = 3
+        self.FOCUS_HEADER_EDIT_BOX = 4
         self.FOCUS_MESSAGE_CONTAINER = 1
-        self.FOCUS_MESSAGE_BODY_BOX = 0
+        self.FOCUS_MESSAGE_BODY_BOX = 1
```

Comment:
I'm surprised we need to bump this value up by one? We're not introducing a column in the the message container?

Reply:
Yeah, that'll be fixed in updated commit :)